charts/php/templates/cronjob.yaml

```diff
@@ -37,10 +37,10 @@ spec:
         spec:
           serviceAccountName: {{ include "php.application.name" . }}
           containers:
-            - name: {{ include "php.application.name" . }}
-              image: {{ include "php.application.imageURL" . }}
+            - name: "{{ include "php.application.name" . }}"
+              image: "{{ include "php.application.imageURL" . }}"
               imagePullPolicy: {{ .Values.application.image.pullPolicy }}
-              args: ["{{ .Values.cron.args }}"]
+              args: [{{ .Values.cron.args }}]
```

Comment:
I think it should be fine to leave the `"` here. Take a look at the node migration job: https://github.com/deliowales/helm-charts-public/blob/c01acf22fa60595cf8d1c1f261585c8c36618268/charts/node/templates/migration-job.yaml#L29